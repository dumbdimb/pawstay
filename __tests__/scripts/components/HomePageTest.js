jest.unmock('../../../src/scripts/components/HomePage');

import React from 'react';
import renderer from 'react-test-renderer';

import HomePage from '../../../src/scripts/components/HomePage';

describe('HomePage', () => {
    it('renders correctly', () => {
        expect(renderer.create(<HomePage/>).toJSON()).toMatchSnapshot();
    });
});