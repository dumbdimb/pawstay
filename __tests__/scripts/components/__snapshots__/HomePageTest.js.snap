// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage renders correctly 1`] = `
<div
  className="home"
>
  <div
    className="home__blurb"
  >
    <div
      className="home__blurb__hello"
    >
      hello
    </div>
    <div
      className="home__blurb__who-we-are"
    >
      we are
    </div>
    <div
      className="home__blurb__who-we-are"
    >
      a personal owned and home based
    </div>
    <div
      className="home__blurb__who-we-are"
    >
      dog boarding and daycare
    </div>
  </div>
  <div
    className="home__darby"
  />
</div>
`;
